### DISCUSSION TITLE
INTERNAL_SERVER_ERROR for gatsby-source-shopify when running "gatsby develop"

### DISCUSSION THREAD

**Discussion by usaamatahir** [gatsbyjs/gatsby#35945] [Help]
Hi everyone, I'm working on a gatsby Shopify build. The Shopify store has almost 250 products. When I run the `gatsby develop` command it gives INTERNAL_SERVER_ERROR.

I have worked on some other Shopify builds with gatsby but never faced this type of error. 
Does anyone know what's the issue?

![Capture](https://user-images.githubusercontent.com/66199337/174229272-9551386e-dee6-49fd-9fcb-0538fcb68f00.PNG)

**Comment by sarmadgardezi** [comment]
Have you check package.json file it can be that error. This can also be error while fetching the Shopify API

**Reply by usaamatahir** [reply]
What do I need to check-in Package.json file?
